lib/polariscope.rb

```diff
@@ -1,23 +1,23 @@
 # frozen_string_literal: true
 
 require_relative 'polariscope/version'
-require_relative 'polariscope/scanner/codebase_health_score'
+require_relative 'polariscope/scanner/gemfile_health_score'
 require_relative 'polariscope/scanner/gem_versions'
 require_relative 'polariscope/file_content'
 
 module Polariscope
   Error = Class.new(StandardError)
 
   class << self
-    def scan(gemfile_content: nil, gemfile_lock_content: nil, bundler_audit_config_content: nil)
-      Scanner::CodebaseHealthScore.new(
-        gemfile_content: gemfile_content || FileContent.for('Gemfile'),
-        gemfile_lock_content: gemfile_lock_content || FileContent.for('Gemfile.lock'),
-        bundler_audit_config_content: bundler_audit_config_content || FileContent.for('.bundler-audit.yml')
-      ).health_score
+    def scan(**opts)
```

Comment:
This change will enable us to support command-line arguments for all configuration options later on.